src/components/AddTodoForm/AddTodoForm.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './AddTodoForm.css';
+
+import users from '../../api/users';
+
+export class AddTodoForm extends React.Component {
+  state = {
+    selectedUserId: '',
+    title: '',
+    hasEmptyUserId: false,
+    hasEmptyTitle: false,
+  }
+
+  setUser = (event) => {
+    this.setState({
+      selectedUserId: event.target.value,
+      hasEmptyUserId: false,
+    });
+  };
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+
+    if (!this.state.title && !this.state.selectedUserId) {
+      this.setState({
+        hasEmptyTitle: true,
+        hasEmptyUserId: true,
+      });
+    }
+
+    if (!this.state.title) {
+      return this.setState({
+        hasEmptyTitle: true,
+      });
+    }
+
+    if (!this.state.selectedUserId) {
+      return this.setState({
+        hasEmptyUserId: true,
+      });
+    }
+
+    const newTodo = {
+      userId: users.find(user => user.name === this.state.selectedUserId).id,
+      id: this.props.todos.length + 1,
+      title: this.state.title,
+      completed: false,
+      user: users.find(user => user.name === this.state.selectedUserId),
+    };
+
+    this.props.onAdd(newTodo);
+
+    return this.setState(state => ({
+      selectedUserId: '',
+      title: '',
+    }));
+  }
+
+  handleChange = (event) => {
+    this.setState({
+      title: event.target.value,
+      hasEmptyTitle: false,
+    });
+  }
```

Comment:
```suggestion
  handleChangeTitle = (event) => {
    this.setState({
      title: event.target.value,
      hasEmptyTitle: false,
    });
  }
```
напиши это возле handleSelectUser